rpc/ipc_windows.go

```diff
@@ -360,6 +363,9 @@ func Listen(address string) (*PipeListener, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Ugly hack working around some async connectivity issues
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Since adding back the inproc client made tests possible via that channel, I've switched everything over to inproc and obsoleted the IPC usage in tests. Hence the error no longer was relevant and I reverted these back.

We should definitely get to the bottom of this, just figured we can do it in another PR and leave it off from this one.